types/wordpress__components/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for @wordpress/components 8.5
+// Type definitions for @wordpress/components 9.0
```

Comment:
👍 Looks like this was _deprecated_ in 9.0

https://unpkg.com/browse/@wordpress/components@9.0.0/src/button/index.js